public/styles/base/_actions.scss

```diff
@@ -79,6 +80,7 @@
 }
 
 @mixin button-subtle {
+  @include button-default;
```

Comment:
did we need this for other states? did we test places where button-subtle is used?

Reply:
This should be the only one that needs it since the rest are already inheriting button default. I checked around for unintended changes.